vital/algo/bundle_adjust.cxx

```diff
@@ -62,8 +62,8 @@ ::optimize(
   kwiver::vital::simple_camera_perspective_map &cameras,
   kwiver::vital::landmark_map::map_landmark_t &landmarks,
   vital::feature_track_set_sptr tracks,
-  const std::set<vital::frame_id_t>& fixed_cameras,
-  const std::set<vital::landmark_id_t>& fixed_landmarks,
+  const std::set<vital::frame_id_t>& /* fixed_cameras */,
```

Comment:
Why use VITAL_UNUSED in some places and commenting out in others?

Reply:
Don't know why this was handled as a comment. Has been updated to use VITAL_UNUSED